packages/typography/src/elements/CodeBlock.tsx

```diff
@@ -41,14 +43,20 @@ export const CodeBlock = React.forwardRef<HTMLPreElement, ICodeBlockProps>(
     const containerRef = useRef<HTMLDivElement>(null);
 
     const code = (Array.isArray(children) ? children[0] : children) as string;
-    let _size: 'sm' | 'md' | 'lg';
-
-    if (size === 'small') {
-      _size = 'sm';
-    } else if (size === 'medium') {
-      _size = 'md';
-    } else {
-      _size = 'lg';
+    let _size: SIZE;
+
+    switch (size) {
+      case 'small':
+        _size = 'sm';
+        break;
+
+      case 'medium':
+        _size = 'md';
+        break;
+
+      case 'large':
+        _size = 'lg';
+        break;
```

Comment:
Just for my own curiosity, is there a reason a switch statement is being used instead of a dictionary lookup?

Reply:
There might be, but it isn't a good one 😅. Updated 😉.